Resource Management Console (#25)
funky-station/docs: src/design-proposals/resource-management-console.md

# Resource Overview Console

| Designers | Implemented | GitHub Links |
|---|---|---|
| TropicalOwl | :x: No | TBD |

## Overview

> The removal of the global ore silo from the game was an inevitable change- While massively helpful to solve resource management issues regarding the existence of several lathes and fabricators, it created another massive issue: It completely trivialized and hyper-optimized what's essentially a core duty of the cargo technician: To supply.

Reviewer: ```suggestion
The removal of the global ore silo from the game was an inevitable change. While it was massively helpful to solve resource management issues regarding the existence of several lathes and fabricators, it created another massive issue: It completely trivialized and hyper-optimized the supply of materials across the station.
```

> Now that it is removed it is up to the cargo technicians to keep the station's localized ore silos well-stocked, that does come with a price however, with every lathe and silo essentially being thrown to one of the four corners of the station resource management proves itself very difficult, and it's not uncommon for cargo techs to eventually become weary of resource distribution, causing at times for them to become stingy towards any departments that are not science- And in turn- Denying essential resources.

Reviewer: ```suggestion
Now that it is removed it is up to the cargo technicians to keep the station's localized ore silos well-stocked which does come with a price; with every lathe and silo essentially being thrown to one of the four corners of the station, resource management proves itself very difficult and it's not uncommon for cargo techs to eventually become weary of resource distribution, causing at times for them to become stingy towards any departments that are not science, leading to resource denial across the board.
```

## Background

I recently returned to Funkystation after suffering from a health issue, I needed a SS14 server to play that wasn't as fast-paced, and so this became my home. However I was a bit bummed to notice the techfab of the department I was playing at wasn't stocked at all 2 hours into the shift!

> I've mentioned the issues this could create- Those being the lack of player authority over resource management and simply, difficult resource management in general. While this doesn't solve player authority as an issue it does optimize resource management *enough* for that to likely not be an issue at all to begin with.

Reviewer: ```suggestion
I've mentioned the issues this could create; the lack of player authority over resource management and difficult resource management in general. While this doesn't solve player authority as an issue it does optimize resource management *enough* for that to likely not be an issue at all to begin with.
```

# The new stuff

## The Resource Overview Console

> Think of it as the energy distribution console of the cargo man: It's a simple concept, it gives you an overview of the lathes and ore silos on the station, with the ore silos being your "SMES'es" and the lathes/fabs being your "APC's" or "Substations".

Reviewer: ```suggestion
Think of it as the energy distribution console of the cargo man: it's a simple concept, it gives you an overview of the lathes and ore silos on the station, with the ore silos being your "SMES'es" and the lathes/fabs being your "APC's" or "Substations".
```

> The idea behind this is to allow cargo technicians to peek at which departments are in more need of X or Y material- And blare an alert if a department is currently not stocked with the three essential resources (Steel, Glass and Plastic)

Reviewer: ```suggestion
The idea behind this is to allow cargo technicians to peek at which departments are in more need of X or Y material and blare an alert if a department is currently not stocked with the three essential resources (Steel, Glass and Plastic)
```

Reviewer: If you have departmental techfabs (I haven't checked before posting this) you could add a component that changes the three essential resources because for science, for example, the essential resources also oftentimes include at minimum 30 plasma.
